src/lava/lib/peripherals/dvs/prophesee.py

```diff
@@ -0,0 +1,241 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import sys
+
+try:
+    import metavision_core
+except ImportError:
+    print("Need `metavision` library installed.", file=sys.stderr)
+    exit(1)
+
+import numpy as np
+import time
+
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+from lava.magma.core.model.py.ports import PyOutPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.process.ports.ports import OutPort
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.resources import CPU
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.lib.peripherals.dvs.transform import Compose, EventVolume
+
+from metavision_core.event_io import RawReader, EventDatReader
+from metavision_ml.preprocessing.event_to_tensor import histo_quantized
+
+
+class PropheseeCamera(AbstractProcess):
+    """
+    Process that receives events from Prophesee device and sends them out as a
+    histogram.
+
+    Parameters
+    ----------
+    device: str
+        String to filename if reading from a RAW/DAT file or empty string for
+        using a camera.
+    biases: dict
+        Dictionary of biases for the DVS Camera.
+    filters: list
```

Comment:
list of what?

Reply:
Seems like, they don't have a base class we could use as a type hint.